components/blocks/tripleColumnImageBlock.tsx

```diff
@@ -0,0 +1,116 @@
+import Image from "next/image";
+import { Key } from "react";
+import type { Template } from "tinacms";
+import { isEmpty } from "../training/eventBooking";
+
+export type TripleColumnImageBlockProps = {
+  imageList: ImgCardProps[];
+};
+
+export type ImgCardProps = {
+  title: string;
+  subTitle?: string;
+  altText: string;
+  imageSrc: string;
+  height?: number | undefined;
+  width?: number | undefined;
+};
+
+export const TripleColumnImageBlock: React.FC<TripleColumnImageBlockProps> = ({
+  imageList,
+}) => {
+  return (
+    <div className="grid grid-cols-12 px-4 py-1">
```

Comment:
Can you replicate the behaviour from the v1 site, where it is not a grid anymore on smaller screens

Reply:
Done